cookbooks/bcpc/Berksfile

```diff
@@ -1,5 +1,5 @@
 source 'https://supermarket.chef.io'
 
-cookbook 'cobblerd', git: 'https://github.com/bloomberg/cobbler-cookbook.git'
-cookbook 'bfd', git: 'https://github.com/bloomberg/openbfdd-cookbook'
+cookbook 'cobblerd', git: 'https://github.com/cbaenziger/cobbler-cookbook.git', branch: chef_12
+cookbook 'bfd', git: 'https://github.com/cbaenziger/openbfdd-cookbook', branch: chef_12
```

Comment:
Please take a look at their pull requests ti move them. I believe Lev just merged openbfdd and now cobbler remains.